src/datasource/dashboards/twinmaker-main-dashboard.json

```diff
@@ -307,22 +436,29 @@
       "type": "grafana-iot-twinmaker-sceneviewer-panel"
     },
     {
-      "datasource": null,
+      "datasource": {
+        "type": "datasource",
+        "uid": "grafana"
+      },
       "gridPos": {
         "h": 12,
         "w": 9,
         "x": 15,
-        "y": 9
+        "y": 15
       },
       "id": 12,
       "options": {
-        "componentName": "${sel_video_comp}",
-        "datasource": "${DS_AWS_IOT TWINMAKER}",
```

Comment:
Should we be worried about the removal of the variable brackets for this and the other variables?

Reply:
I tried to understand why this was happening and at least produce something consistent, but I was unsuccessful.  I don't understand why some of them are added and some aren't. 
I also suspect that sel_entity_comp and sel_video_comp might not be set (in the transforms?).

I am reluctant to modify anything manually in the json... so all of this came from the export of the dashboard on v8.4.7. Any ideas?

Reply:
I think it needs to be `${DS_AWS_IOT TWINMAKER}` to be set properly. I tried it with a twinmaker datasource named `TwinMaker` instead of the default `AWS IoT TwinMaker` and the scene viewer panel displayed a `No TwinMaker Data Source Connected` error. I think it's ok for you to edit the json, the other panels still use the bracketed variables, and you can just load it up and make sure it still works as expected.

Reply:
I re-examined the panel and after choosing Default, the datasource is not referred to by it's name anymore. 

I still prefer not to edit the json directly --got this on v8.4.7.

Tested it and this is importing well on v9.3.6 on play.grafana.org